client/cody/webviews/Recipes.tsx

```diff
@@ -16,6 +16,7 @@ export const recipesList = {
     fixup: 'Fixup code from inline instructions',
     'context-search': 'Codebase context search',
     'release-notes': 'Generate release notes',
+    'pr-description': 'Generate PR description',
```

Comment:
I think we should spell this out. What do you think?

```suggestion
    'pr-description': 'Generate pull request description',
```

Reply:
Agreed with you here, let me update this.